packages/flutter_goldens_client/lib/client.dart

```diff
@@ -119,13 +119,39 @@ class GoldensClient {
   }
 
   Future<void> _checkCanSync() async {
+    final bool isCirrus = (platform.environment['CIRRUS_CI'] ?? '').isNotEmpty;
```

Comment:
nit:  I think we check this elsewhwer as just `platform.environment.containsKey('CIRRUS_CI')`